sscanss/editor/dialogs.py

```diff
@@ -5,6 +5,7 @@
 from sscanss.core.math import clamp
 from sscanss.core.util import create_scroll_area
 from .widgets import ScriptWidget, JawsWidget, PositionerWidget, DetectorWidget
+#from sscanss.app.window.view import MainWindow
```

Comment:
please delete rather than comment out this import. Git keeps history so if need later can always be brought back in